app/_includes/insomnia/domain-capture.md

```diff
@@ -1,12 +1,14 @@
-Domain capture allows you to consolidate all Insomnia accounts in a specific domain into your Enterprise account.
-To enable domain capture:
-
+Use domain capture to automatically add new users to your Insomnia Enterprise account when they sign up with a verified email domain.
+To enable domain capture and auto-capture all new employees:
 1. Add and verify a [domain](https://app.insomnia.rest/app/enterprise/domains/list).
 1. In the domain settings, click the toggle to enable domain capture.
 
 {:.info}
-> Domain capture can only be enabled on a verified domain.
+> You can enable domain capture on a verified domain only.
+
+When you enable domain capture on a verified domain, all new users that create an Insomnia account with an email address in the domain are automatically added to the Enterprise account. For example:
 
-Once domain capture is enabled, when a new user creates an Insomnia account with an email address in the domain, the user will be added to the enterprise account automatically.
+- Kong uses `@kong.com` and enables domain capture: When Amal@kong.com signs up for Insomnia, the platform automatically adds Amal@kong.com to the Enterprise license and lists the user on the licenses page.
```

Comment:
@TuannasCodeCatastrophe @Guaris 

We don't own `kong.com` as a domain. In general, we should use `@example.com` in examples as the domain exists specifically for documentation

For names, I recommend sticking with `alice`, `bob` and `charlie` for consistency with the rest of the docs